server/db/model/Event.js

```diff
@@ -1,4 +1,65 @@
-/* eslint-disable */
 var mongoose = require('../index.js'),
-    config = require('../../../config/default.js'),
-    Schema = mongoose.Schema;
+    Schema = mongoose.Schema,
+    Location = require('./Location.js');
+
+// Define the document Schema
+var schema = new mongoose.Schema({
+    title: String,
+    desc: String,
+    startDate: {
+        type: Date,
+        default: Date.now()
+    },
+    endDate: {
+        type: Date,
+        default: Date.now()
+    },
+    category: {
+        type: String,
+        enum: ['General', 'Food', 'Tech Talk', 'Sponsor Event', 'Other'],
+        default: 'General'
+    },
+    location: {
+        type: Schema.Types.ObjectId
```

Comment:
can you add a ref property to 'Location' here:

`location: {
    type: Schema.Types.ObjectId,
    ref: 'Location'
}`